threads-csharp/TaskVsThread/TaskVsThread/CarBuildingWithThread.cs

```diff
@@ -22,7 +22,7 @@ public static void Build(int horsePower, int suspensionCount)
         });
         engineThread.Start();
 
-        ConcurrentBag<Suspension> suspensions = new ConcurrentBag<Suspension>();
+       var suspensions = new ConcurrentBag<Suspension>();
```

Comment:
This should have another space to be properly aligned, sorry super picky!

Reply:
Sorry, I don't know how I messed up the indentation